coi-client-spec.md

```diff
@@ -1367,4 +1367,4 @@ A COI-compliant folders generates these folders automatically when a client call
 The security consideration of the referenced standards also apply to this COI client specification definition.
 
 # IANA Considerations
-The "pinned" value of the "Content-Disposition" header field should be registered with IANA. 
\ No newline at end of file
+The "pinned" value of the "Content-Disposition" header field should be registered with IANA. 
```

Comment:
Didn't edit this line. Maybe github's web-editor did this?

Reply:
Many thanks!